rhinosystem/window.ui

```diff
@@ -38,11 +38,26 @@
           </object>
         </child>
         <child>
-          <object class="GtkLabel" id="version">
+          <object class="GtkButton" id="version_invalid">
             <property name="margin-top">5px</property>
+            <property name="halign">center</property>
             <property name="label">Failed to fetch version</property>
             <style>
-              <class name="dim-label"/>
+              <class name="error"/>
+              <class name="pill"/>
+              <class name="flat"/>
+            </style>
+          </object>
+        </child>
+        <child>
+          <object class="GtkButton" id="version">
+            <property name="margin-top">5px</property>
+            <property name="halign">center</property>
+            <property name="label">no_version_set</property>
+            <style>
+              <class name="accent"/>
+              <class name="pill"/>
+              <class name="flat"/>
```

Comment:
The `flat` style should be removed, since there is no way for users to see that they can click on the version code with it.

Reply:
Without flat, the button will pop out too much (in my opinion)

Reply:
That is what the `accent` class is supposed to do
it would also be possible to make a custom css class that make the background more opaque, but the button will need a background

Reply:
Imo it should be kept as a pill 24/7, it looks good and really is better.

Reply:
Removed flat from the version pill

Reply:
You only removed it from the `version_invalid` button, not the `version` one.

Reply:
Removed from both now